pokete_data/map_data.py

```diff
@@ -1,14 +1,14 @@
 map_data = {
-#     "playmap_1": {
-#         "hard_obs": {
-#         },
-#         "soft_obs": {
-#         },
-#         "dors": {
-#         },
-#         "balls": {
-#         }
-#     },
+    #     "playmap_1": {
```

Comment:
Why is this indented?

Reply:
Comments should be indented the same width as their scope, in this case the following "real" playmap_1